webservices/common/views.py

```diff
@@ -78,26 +78,23 @@ def get(self, **kwargs):
             kwargs["two_year_transaction_period"] = range(
                 1976, utils.get_current_cycle() + 2, 2
             )
```

Comment:
Instead of doing
```
query = self.build_query(**kwargs)	
count, _ = counts.get_count(self, query)
```
three times, I did it at the beginning for all and then generated a new `query` for the `UNION ALL` scenarios.